deps/wazuh_testing/wazuh_testing/modules/fim/utils.py

```diff
@@ -319,6 +319,20 @@ def transform_registry_list(value_list=['test_value'], value_type=fim.REG_SZ, ca
         return aux_dict
 
 
+def transform_registry_key_list(key_list=['test_value'], callback=ev.callback_key_event):
+    if sys.platform == 'win32':
```

Comment:
Because we do not want the function to do anything in other OSs. Hence with this, we can define empty functions for `linux`, `darwin` and `sunos` platforms